### PR TITLE
 PIL-1256 Revamp fiat on ramp - Add Cash

### PR DESCRIPTION
**Pull request by divyaBacancy**
https://user-images.githubusercontent.com/66996635/153208644-90423e60-92be-4e74-84e4-25051b87aaba.mov

### CODE REVIEW

**Review by Yash-Chaubal — COMMENTED** [review]
Looking good, great work! Could you just make sure new files are .tsx rather than .js please?

File: src/screens/AddCash/AddCash.js [hunk 1]
```diff
@@ -62,48 +64,65 @@ const AddCash = () => {
...
+      setCurrencySymbol('$');
+    }
+  };
+
+  const networkSelected = () => {
+    if (inUS) {
+      openWert();
+    } else {
+      openRamp();
+    }
```
Comment (Yash-Chaubal) [inline src/screens/AddCash/AddCash.js:83]: Could some of these if statements also be single line, like so:
```suggestion
    if (inUS) openWert();
    else openRamp();
```

Comment (Yash-Chaubal) [inline src/screens/AddCash/AddCash.js:83]: Could some of these if statements also be single line, like so:
```suggestion
    if (inUS) openWert();
    else openRamp();
```

**Comment by divyaBacancy** [comment]
@Yash-Chaubal Thanks for reviewing. Done with the changes.

**Review by Yash-Chaubal — APPROVED** [review]
LGTM!

**Pull request merged**